pkg/lca/imagebasedupgrade.go

```diff
@@ -223,6 +223,23 @@ func (builder *ImageBasedUpgradeBuilder) WithSeedImage(
 	return builder
 }
 
+// WithOadpContent adds oadpContent to be used by the imagebasedupgrade.
+// This is used for backup/restore during upgrade.
+func (builder *ImageBasedUpgradeBuilder) WithOadpContent(
+	oadpConentConfigMapName string, oadpConentConfigMapNamespace string) *ImageBasedUpgradeBuilder {
```

Comment:
```suggestion
	oadpContentConfigMapName string, oadpContentConfigMapNamespace string) *ImageBasedUpgradeBuilder {
```

Reply:
@achuzhoy re: https://github.com/openshift-kni/eco-goinfra/pull/235#discussion_r1474633224, it's just a typo

Reply:
conent vs content

Reply:
Updated. Thanks.